src/Services/ModulePermissionHandler.php

```diff
@@ -35,6 +37,280 @@ class ModulePermissionHandler {
    */
   protected $configFactory;
 
+  /**
+   * The entity type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  public const BASE_SB_PERMISSIONS = [
```

Comment:
SB is Site Builder? I'd prefer using the full name to avoid confusion, but that can be adjusted in the future, and doesn't need to hold this up.